src/components/Navbar/DeskTopNavigation.tsx

```diff
@@ -0,0 +1,183 @@
+import { DownOutlined, UpOutlined } from '@ant-design/icons'
+import { t, Trans } from '@lingui/macro'
+import { Dropdown, Menu, Space } from 'antd'
+import Link from 'next/link'
+import { CSSProperties, useEffect, useState } from 'react'
+
+import Logo from './Logo'
+import { navMenuItemStyles, topNavStyles, topRightNavStyles } from './navStyles'
+
+import { Header } from 'antd/lib/layout/layout'
+import Account from './Account'
+import { resourcesMenuItems } from './constants'
+import { TransactionsList } from './TransactionList'
+
+const resourcesMenu = (
+  <Menu
+    items={resourcesMenuItems()}
+    style={{ marginTop: -16, marginLeft: -6 }}
+  />
+)
+
```

Comment:
this file needs to be moved to https://github.com/jbx-protocol/juice-interface/pull/2234/files


and `TopLeftItems` should be deleted since it's a rename.